File: 8Desempeño/Numericos.c
//gcc Numericos.c -lm -o num1
#include <math.h>
#include <stdio.h> 

int main(){
    double max = 1757600.0; 
    double seno,cose,tang,logn,raiz;
    double i = 0.0;
    while(i < max){
        seno += sin(i);
        cose += cos(i);
        tang += tan(i);
        logn += log(i);
        raiz += sqrt(i);
        i++;
    }
    return 0;
}

File: 8Desempeño/Numericos.py
import math
import time

def main():
    max = 1757600.0
    calculosNumericos(max)

def calculosNumericos(m):
    max = float(m)
    i = 0.00
    seno = 0.00
    cos = 0.00
    tan = 0.00
    log = 0.00
    raiz = 0.00
    while i < max:
        seno += math.sin(i)
        cos  += math.cos(i)
        tan +=  math.tan(i)
        log += math.log1p(i)
        raiz += math.sqrt(i)
        i +=1

main()